### DISCUSSION TITLE
Automation website flow

### DISCUSSION THREAD

**Discussion by yanirmanor** [johnlindquist/kit#319] [General]
Is it possible to create a script that will run playwright/puppeteer for website automation flow?

**Comment by johnlindquist** [comment]
Yah, for sure!

Here's a hello world example. I disabled "headless" so you can see it working.

[Install playwright-test](https://scriptkit.com/api/new?name=playwright-test&url=https://gist.githubusercontent.com/johnlindquist/7882786b5fb9fd0b1897fc9e7bd8befb/raw/ebdb5c40897a531fbc7e86935c60a8afb19ec1ae/playwright-test.js)

```js
/** @type typeof import("playwright")) */
let { webkit } = await npm("playwright")

let screenshotPath = tmp(`screenshot.png`)

let browser = await webkit.launch({ headless: false })
let page = await browser.newPage()
await page.goto("http://whatsmyuseragent.org/")
await page.screenshot({ path: screenshotPath })

await browser.close()

exec(`open "${screenshotPath}"`)

```
